js/personal.js

```diff
@@ -94,19 +106,42 @@ View.prototype = {
             OC.Notification.showTemporary(t('facerecognition', 'There was an error trying to show your friends'));
         });
     },
+    setEnabledUser: function (enabled) {
+        var self = this;
+        $.ajax({
+            type: 'GET',
+            url: OC.generateUrl('apps/facerecognition/setuservalue'),
+            data: {
+                'type': 'enabled',
+                'value': enabled
+            },
+            success: function () {
+                if (enabled) {
+                    OC.Notification.showTemporary(t('facerecognition', 'The analysis is enabled, please be patient, you will soon see your friends here.'));
+                } else {
+                    OC.Notification.showTemporary(t('facerecognition', 'The analysis is disabled, we eliminate all information for the recognition of your friends.'));
```

Comment:
eliminate -> eliminated? I would even go with "removed all face data"

Reply:
I think you already suggested it in another branch.. :sweat_smile: 
Ok.